src/scripts/main.js

```diff
@@ -354,7 +354,36 @@ const people = [
   },
 ];
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const table = document.querySelector('.dashboard');
 
-// write your code here
+function createRow(person) {
+  const row = document.createElement('tr');
+
+  const nameCell = document.createElement('td');
+  const genderCell = document.createElement('td');
+  const bornCell = document.createElement('td');
+  const diedCell = document.createElement('td');
+  const ageCell = document.createElement('td');
+  const centuryCell = document.createElement('td');
+
+  nameCell.textContent = person.name;
+  genderCell.textContent = person.gender;
+  bornCell.textContent = person.born;
+  diedCell.textContent = person.died;
+  ageCell.textContent = person.died - person.born;
+  centuryCell.textContent = Math.ceil(person.died / 100);
+
+  row.appendChild(nameCell);
+  row.appendChild(genderCell);
+  row.appendChild(bornCell);
+  row.appendChild(diedCell);
+  row.appendChild(ageCell);
+  row.appendChild(centuryCell);
+
+  return row;
+}
+
+people.forEach(person => {
+  const row = createRow(person);
+  table.appendChild(row);
+});
```

Comment:
Appending rows to the table inside a loop causes the browser to reflow the document multiple times, which can lead to performance issues. It's more efficient to build the entire table in memory and then append it to the DOM once at the end. You could create a document fragment, append your rows to that, and then append the fragment to the table.